resources/setup_emulator.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+
+kill_all_emus() {
```

Comment:
^ when does this function get called?

Reply:
It will be called from 'execute_testing.sh` after execution is finished.

Reply:
If the function is called in a different script, it should be in that script. However, I think it makes sense to be used in this script. I did a quick search, and it seems this script is not called yet.

Reply:
I've called this function before creating new emulator, plus it will be needed to call that function after execution is finished as well.